src/creature.h

```diff
@@ -185,6 +185,11 @@ class Creature
         dealt_projectile_attack projectile_attack( const projectile &proj, const tripoint &target,
                                                    double total_dispersion );
 
+        /**
+         * Return the chance that a projectile attack will hit with at least the given accuracy
```

Comment:
Right, my point was that the current style doesn't match what you're requesting. I'll add the javadoc-style tags, they are good to have, but I'm not psychic and elsewhere you have been unhappy when I have not followed the style of the surrounding code.